### DISCUSSION TITLE
When to decide a variable is not a probability

### DISCUSSION THREAD

**Discussion by wickedtuna04** [MSKCC-Epi-Bio/decisioncurveanalysis#20] [Q&A]
Hello! the software tutorial has been super helpful so thank you for that :) 

Referencing this line " For the model itself, we would actually need to first specify that the marker variable – unlike those of any of the models before – is not a probability. " under the section "Saving out Net Benefit Values" I was wondering when a variable is considered not a probability as I note that in this tutorial, 'Marker' was considered not a probability while 'Family History' is  >> and code wise this would include adding the line 'as_probability = 'marker' ' >> which would generate an output where the net benefit of the variable is always the same as that of 'treat all' until the net benefit of the variable exceeds that of 'treat all' at a certain threshold (I am referencing the table in this same section right under the line i quoted above) 

Would this be because 'Family History' is a binary variable while 'Marker' is not? 

I am performing a DCA for: imaging modality vs biopsy all, where the imaging modality is a binary variable (positive on imaging vs. negative on imaging), I am therefore wondering if I should be considering imaging modality a probability >> this affects when the net benefit of imaging exceeds that of biopsy all >> which in turns affects my calculation on net biopsies avoided by performing imaging prior to deciding who goes for biopsy! 

To use the example in the tutorial (referring to the table on net benefits): If 'marker' were a probability >> the net benefit of 'marker' at 5% is actually 0.015 ie less than that of 'treat all' >> translating this to net interventions avoided would thus give a negative value (rather than 0 if 'marker' were not a probability) >> which if I understand correctly would mean using the marker at threshold 5% would result in necessary interventions being missed (ivo the negative value). 

any help on this would be greatly appreciated as i would not want to misuse DCA for my analyses and misrepresent my data O.O 

<img width="451" alt="Picture 1" src="https://github.com/MSKCC-Epi-Bio/decisioncurveanalysis/assets/163866057/caf2ea68-0d73-406e-9cd0-2e6a158904c1">
<img width="451" alt="Picture 2" src="https://github.com/MSKCC-Epi-Bio/decisioncurveanalysis/assets/163866057/50f5a5ae-f301-437e-bf72-0ecc8e6b74ba">
<img width="444" alt="Picture 3" src="https://github.com/MSKCC-Epi-Bio/decisioncurveanalysis/assets/163866057/20f776ce-30b4-4436-b7fb-60d15f88b0be">

**Comment by VickersA** [comment]
Family history and other binary variables are either 0 or 1, so are considered probabilities. Consider a standard diagnostic test: the doctor say "your test came back positive, you'll need surgery" or "the good news is that the test is negative, so no need to operate right now". A marker is not on a probability scale but will be given in units such as ng / mL. In this case, the marker varies from close to zero to above 6.

**Reply by wickedtuna04** [reply]
Thank you for the help Dr Vickers!!
